app/views.py

```diff
@@ -226,16 +227,14 @@ def getSingleRequest(currentUser, requestid):
         response.status_code = 405  # Method not allowed
```

Comment:
405 is not the right status code for this, raise a 400 if it is an error in user input or 404 if it is not found